src/components/headline.stories.tsx

```diff
@@ -0,0 +1,96 @@
+// ----- Imports ----- //
+
+import React, { ReactElement } from 'react';
+import { withKnobs, select, boolean, radios } from '@storybook/addon-knobs';
+
+import Headline from './headline';
+import { Item, Design, Display } from 'item';
+import { Pillar } from 'pillar';
+import { None } from 'types/option';
+
+
+// ----- Setup ----- //
+
+const item: Item = {
+    pillar: Pillar.news,
+    design: Design.Article,
+    display: Display.Standard,
+    body: [],
+    headline: 'Reclaimed lakes and giant airports: how Mexico City might have looked',
+    standfirst: new None(),
+    byline: '',
+    bylineHtml: new None(),
+    publishDate: new None(),
+    mainImage: new None(),
+    contributors: [],
+    series: {
+        id: '',
+        type: 0,
+        webTitle: '',
+        webUrl: '',
+        apiUrl: '',
+        references: [],
+    },
+    commentable: false,
+    tags: [],
+};
+
+const pillarOptions = {
+    News: Pillar.news,
+    Opinion: Pillar.opinion,
+    Sport: Pillar.sport,
+    Culture: Pillar.arts,
+    Lifestyle: Pillar.lifestyle,
+};
+
+const starRating: Record<number, number> = [1, 2, 3, 4, 5];
```

Comment:
I think we should be able to handle 0 (even if there hasn't been any in the last 200 reviews)

Reply:
👍 updated

Is it possible for a review to be given 0 stars?

Reply:
It might not be (I overheard someone saying there hasn't been in the last 200 reviews), but I think it could possibly change one day, so we might as well handle it?